account-kit/core/src/store/store.ts

```diff
@@ -273,6 +273,9 @@ const addClientSideStoreListeners = (store: Store) => {
         }));
       });
 
+      // TODO: handle this appropriately
+      // signer.on("newUserSignup", () => console.log("got new user signup"));
```

Comment:
we don't need to handle this in this PR

We should be emitting this event from core and subscribing to it from react instead of skipping core directly in react, but we can solve that later. Right now, the only way for someone to subscribe to a new user signup is by knowing the signer emits this event. We should add a `watchNewUserSignup` action within core that we can use to subscribe to a signup event

We should update the comment here to point to this discussion on github

Reply:
link added in comment!